katas/main/src/pokerhands/joaquin/Deck.cs

```diff
@@ -0,0 +1,45 @@
+namespace PokerHand.Joaquin;
+
+using System.Collections.Generic;
+
+public class Deck
+{
+    public string[] cards = new string[52];
+
+    public char[] suits = { 'C', 'D', 'H', 'S' };
+
+    public char[] cardValues = { '2', '3', '4', '5', '6', '7', '8', '9', 'T', 'J', 'Q', 'K', 'A' };
+
+    public Dictionary<char, int> valuesRanking = new Dictionary<char, int>
+    {
+        {'2', 0},
+        {'3', 1},
+        {'4', 2},
+        {'5', 3},
+        {'6', 4},
+        {'7', 5},
+        {'8', 6},
+        {'9', 7},
+        {'T', 8},
+        {'J', 9},
+        {'Q', 10},
+        {'K', 11},
+        {'A', 12},
+    };
+
+    public Deck()
+    {
+        int index = 0;
+        while (index < 52)
```

Comment:
Maybe this `while` is not needed? both `for` blocks will iterate 52 times anyway, since `suits` size is 4 and `cardValues` size is 13.

Reply:
Yes, you're right, is not necessary. I'll remove it. Thanks for the recommendation 🙏🏼